### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by restrainedimages** [fetlife/fetlife-android#30]
The latest version crashes.
Once on initial startup.
Then again after login.
Restarting crashes immediately.

Uninstall / reinstall resets the cycle.

Running Nexus 6 running latest version of Android.

(Additional feature request - can we have download links to current and last versions in case of issues like this.)

**Comment by restrainedimages** [comment]
Interestingly it still notified me of a new message. But when I selected the notification it also crashes.

**Comment by Bryan76** [comment]
I have the exact smae experience.  Crashed once on initial setup.  Crashes after login.  And now crashes immediately whenever I try to run the app.

**Comment by kistitan** [comment]
Hi guys,

May I ask if you followed our installation guide?
It is essential to uninstall he previous version of the app before installing the new one.

There were some incompatible database change between the old and the new version.
This changes could be solved without forcing you to uninstalling the old version but it would take some time and we could deliver the new features only with delays.
Of course after the first stable version (1.0) you won't need to uninstall he previous version any more.

Could you please let me know, if uninstalling and reinstalling the app solves the issue you face?

Thanks a lot or your help and patience.

**Comment by restrainedimages** [comment]
When I originally updated I forgot and tried to install over the old one but it failed to install. So I uninstalled / reinstalled.

I've tried force stopping the app and clearing the data - same results.
I've tried uninstalling / reinstalling - same results.

Very reproducible issue, at least for me. Let me know if there is anything I can do to help diagnose.

I'm agree with your current approach. Speed of delivering new features over installation inconvenience for now.
